packages/extension-polkagate/src/popup/staking/partial/NominationsBackButton.tsx

```
@@ -0,0 +1,70 @@
+}
+
+export default function NominationsBackButton ({ style }: Props) {
+  const { t } = useTranslation();
+  const theme = useTheme();
+
+  const containerRef = useRef(null);
+  const hovered = useIsHovered(containerRef);
+  const navigate = useNavigate();
+  const { genesisHash } = useParams<{ genesisHash: string }>();
+
+  const onBack = useCallback(() => navigate('/solo/' + genesisHash) as void, [genesisHash, navigate]);
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)